content/blog/cdf-intro-uber-crane/index.md

```diff
@@ -93,6 +93,6 @@ By adopting OpenFABR CDF (our OSS project) and FABR Infrastructure (our SaaS pro
 
 - It becomes drastically easier to develop, test, and package infrastructure components in such a mixed infrastructure environment. CDF is designed to make this task easier for infrastructure package authors.
 - The standardised configuration format defined by CDF's modular approach not only makes config and change management easier, but also offers a free abstraction layer between organisation-wide shared infrastructure platforms and developers in the same organisation.
-- Developer self-serve wasn't stressed enough. With the adoption of FABR Infra, developer experience around infrastructure self-service is drastically improved. Developers can easily discover and use top-level infrastructure components and services made available to them.
```

Comment:
I feel we should start with and even only speak about what we *do* instead of what others *don't* here.

Reply:
self-serve was a point you had iirc. I don't think we should cut but good call to not be critical in this case.

Reply:
It's still there, just not starting with a criticism on them :)